packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -8,6 +8,18 @@ import 'package:flutter/painting.dart';
 
 export 'dart:ui' show TextDirection;
 
+/// Determines the politeness setting of aria live in Flutter web.
+///
+/// It is used to set the priority with which screen reader should treat updates to live regions.
+enum AriaLivePolitenessSetting {
```

Comment:
The framework is meant to be cross-platform, but the term "ARIA" is web-specific. Also, the fact that on the web we use a live region is an implementation detail. The intent of the API is to produce an announcement. I would therefore avoid using "ARIA" or "live" in the public API and in the API docs. In fact, perhaps just `Politeness` is sufficient. Alternatively, we could invert it, and use `Assertiveness` instead.

Since this is a framework-side API concern, let's loop in @chunhtai and @goderbauer.